pep-0315.txt

```diff
@@ -102,7 +102,7 @@ belongs, and does not require code to be duplicated.
 Syntax
 ======
 
-The syntax of the while statement::
+The syntax of the while statement ::
```

Comment:
Yes, it removes a colon from rendered text (but still makes following indented text a code block). The original text/plain version didn't have a colon, and this looks correct to me. "The syntax of the while statement ... is extended as follows: ..." (where "..." are code blocks) is a single sentence, and it shouldn't have a colon after "statement".